edumfa/api/lib/utils.py

```diff
@@ -287,17 +299,23 @@ def get_all_params(request):
     body = request.data
     return_param = {}
     if param:
-        log.debug("Update params in request {0!s} {1!s} with values.".format(request.method,
-                                                                             request.base_url))
+        log.debug(
+            "Update params in request {0!s} {1!s} with values.".format(
+                request.method, request.base_url
+            )
+        )
         # Add the unquoted HTML and form parameters
         return_param = check_unquote(request, request.values)
 
     if request.method == "GET" and request.form:
         return_param.update(check_unquote(request, request.form))
 
     if request.is_json:
-        log.debug("Update params in request {0!s} {1!s} with JSON data.".format(request.method,
-                                                                                request.base_url))
+        log.debug(
```

Comment:
Seems like flynt did not catch everything...

Reply:
Yeah unfortunately it didn't catch everything. About 400-500 possible occurrences of old string formatting are still present. Not sure if it's worth to update it manually.